test/testdata/fixtures/gc/3.11-jdk11-CMS/k8ssandra.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: k8ssandra.io/v1alpha1
```

Comment:
that fixture name is incorrect (unless it's on purpose and there's something I'm missing).
3.11 can only work with jdk8, which is what's shipping in the images we publish and use.
Shouldn't it be `3.11-jdk8-CMS` instead?